# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/argp/hotfix.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- argp-standalone-1.3/argp-parse.c.vanilla	2021-10-07 18:33:04.934236620 +0200
+++ argp-standalone-1.3/argp-parse.c	2021-10-07 18:33:49.725239814 +0200
@@ -1277,13 +1277,13 @@
 /* Defined here, in case a user is not inlining the definitions in
  * argp.h */
 void
-__argp_usage (__const struct argp_state *__state) __THROW
+__argp_usage (__const struct argp_state *__state) // __THROW
 {
   __argp_state_help (__state, stderr, ARGP_HELP_STD_USAGE);
 }
 
 int
-__option_is_short (__const struct argp_option *__opt) __THROW
+__option_is_short (__const struct argp_option *__opt) // __THROW
 {
   if (__opt->flags & OPTION_DOC)
     return 0;
@@ -1297,7 +1297,7 @@
 }
 
 int
-__option_is_end (__const struct argp_option *__opt) __THROW
+__option_is_end (__const struct argp_option *__opt) // __THROW
 {
   return !__opt->key && !__opt->name && !__opt->doc && !__opt->group;
 }
